apps/website/src/pages/about/tech/commands.tsx

```diff
@@ -61,6 +61,10 @@ const AboutTechPage: NextPage = () => {
     ),
   });
 
+  const [selectedCamera, setSelectedCamera] = useState<keyof typeof presets>(
```

Comment:
Instead of using `keyof typeof presets` here, we should make `const presets` in `presets.ts` an actual `as const satisfies Record<string, Camera>`, and then you can `export type Preset = keyof typeof presets` (this'll collide with the current `interface Preset` which can probably become `interface PresetData` or smth), which can then be used here